ksqldb-parser/src/test/java/io/confluent/ksql/parser/AstBuilderTest.java

```diff
@@ -866,4 +868,17 @@ public void shouldFailOnBigIntWithParameter() {
             () -> givenQuery("CREATE STREAM INPUT (K BIGINT (KEY)) WITH (kafka_topic='input',value_format='JSON');")
     );
   }
+
+  @Test
+  public void shouldSupportHeadersColumns() {
+    // Given:
+    final SingleStatementContext stmt
+        = givenQuery("CREATE STREAM INPUT (K BIGINT HEADERS) WITH (kafka_topic='input',value_format='JSON');");
```

Comment:
This statement raises interesting questions:
- should we allow a "header only" schema definition (I think, yes, why not?)
- if yes, why would we need a `value_format` property for this case?

Reply:
That is a good point. Right now, [all sources](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/create-stream/#stream-properties) sources require the key_format and value_format properties, even if it doesn't actually have a key/value. It would be nice if they are not needed, but that's a separate issue to fix.